contentcuration/contentcuration/frontend/accounts/pages/ActivationSent.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+
+  <ImmersiveModalLayout
+    previousPage="/"
```

Comment:
these should be implemented as named links (e.g. `:to="{ name='Main' }")`) but I'm not sure offhand how to do that with vuetify buttons